includes/api.php

```diff
@@ -53,21 +53,7 @@ class Api {
 	 */
 	private static $api_feedback_url = 'https://my.elementor.com/api/v1/feedback/';
 
-	/**
-	 * Get info data.
-	 *
-	 * This function notifies the user of upgrade notices, new templates and contributors.
-	 *
-	 * @since 2.0.0
-	 * @access private
-	 * @static
-	 *
-	 * @param bool $force_update Optional. Whether to force the data retrieval or
-	 *                                     not. Default is false.
-	 *
-	 * @return array|false Info data, or false.
-	 */
-	private static function get_info_data( $force_update = false ) {
```

Comment:
Why remove the phpDocs?

Reply:
It's so useless, we should remove all other comments from our code :D